geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/DescribeConnectionCommandIntegrationTest.java

```diff
@@ -21,13 +21,12 @@
 import org.junit.experimental.categories.Category;
 
 import org.apache.geode.internal.logging.LogService;
-import org.apache.geode.test.junit.categories.GfshTest;
 import org.apache.geode.test.junit.categories.IntegrationTest;
 import org.apache.geode.test.junit.rules.GfshCommandRule;
 import org.apache.geode.test.junit.rules.GfshCommandRule.PortType;
 import org.apache.geode.test.junit.rules.LocatorStarterRule;
 
-@Category({IntegrationTest.class, GfshTest.class})
+@Category({IntegrationTest.class})
```

Comment:
I think `GfshTest` would be appropriate here. The connection in this test is the gfsh connection to a locator.